genie-common/src/main/java/com/netflix/genie/common/util/GenieDateFormat.java

```diff
@@ -34,6 +41,11 @@
  */
 public class GenieDateFormat extends StdDateFormat {
 
+    /**
+     * Pattern to validate (e.g., in tests) the expected date format this class produces.
+     */
+    @VisibleForTesting
+    public static final Pattern VALID_PATTERN = Pattern.compile("^\\d{4}-\\d{2}-\\d{2}T\\d{2}:\\d{2}:\\d{2}.\\d{3}Z$");
```

Comment:
does it need to be public or could you just make it package protected?

Reply:
It's used from the `web` module/package.